Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -666,13 +666,16 @@ internal nfloat GetEstimatedRowHeight(UITableView table)
 				}
 
 				// We're going to base our estimate off of the first cell
-				Cell firstCell;
 				var isGroupingEnabled = List.IsGroupingEnabled;
 
 				if (isGroupingEnabled)
-					firstCell = templatedItems.ActivateContent(0, templatedItems.GetGroup(0)?.ListProxy[0]);
-				else
-					firstCell = templatedItems.ActivateContent(0, templatedItems.ListProxy[0]);
+					templatedItems = templatedItems.GetGroup(0);
+
+				object _firstCell = null;
+				if (templatedItems?.ListProxy.TryGetValue(0, out _firstCell) != true)
```

Comment:
I was stuck on this one for a moment, wondering if it was adding a cast to `bool?` or not. turns out that Roslyn does not (not tested on other compilers).

But what is really confusing, is that the behavior of this changes if you change `!= true` to `== false`.

As I'm not smart enough to figure that if I ever have to debug that part due to a rogue refactoring, I recommend changing this to
```
if (templatedItems == null || !templatedItems.ListProxy.TryGetValue(0, out _firstCell))
```